src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -700,17 +700,6 @@ void AsyncCleaner::close_segment(segment_id_t segment)
   INFO("closed, {} -- {}", gc_stat_printer_t{this, false}, seg_info);
 }
 
-AsyncCleaner::trim_alloc_ret AsyncCleaner::trim_alloc(
-  Transaction &t,
-  journal_seq_t limit)
-{
-  return backref_manager.merge_cached_backrefs(
-    t,
-    limit,
-    config.rewrite_backref_bytes_per_cycle
-  );
-}
-
 double AsyncCleaner::calc_gc_benefit_cost(
```

Comment:
Nothing is logically changed by the commit name "cleanup".

I think it is clearer to unsplit the straitforward logic about trimming. `gc_trim_alloc()` and `gc_trim_dirty()` shares similar procedure, so also make their logs consistent with better information for debugging.

Further, we might need to rearrange these cleaner transactions so that trim and reclaim won't be concurrent at the higer level with device tiers.